package.json

```diff
@@ -45,6 +45,7 @@
     "husky": "^9.1.7",
     "lint-staged": "^16.1.0",
     "listr2": "^8.0.0",
+    "minimist": "^1.2.8",
```

Comment:
`minimist` is used in `scripts/utils/parsearguments.mjs`, but wasn't added to dependencies. It probably worked when using Yarn (due to hoisting) but no longer works with pnpm.